test/passing/shortcut_ext_attr.ml

```diff
@@ -0,0 +1,147 @@
+(* TEST flags = "-dparsetree" ocamlc_byte_exit_status = "2" *
+   setup-ocamlc.byte-build-env ** ocamlc.byte *** check-ocamlc.byte-output *)
```

Comment:
?

Re `-dparsetres`, note that `ocamlformat -g test.ml -o test.fmt.ml` will dump `test.old.ast` (and `test.new.ast` if different) using `Pprintast` (but after being normalized).

Reply:
This file was taken untouched from the OCaml testsuite.